### PR TITLE
import vs require

### LINKED ISSUE #7400: [Create New Issue]: Require vs Import and Dynamic Imports
### Description

explain differences between require and import syntax and then explain what dynamic importing is in javascript

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by HarjasSodhi**
<hr>

## Description 📜

Added documentation for import vs require
Fixes #7400

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/Javascript/Require vs Import/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+We can see from the above code, that we are importing the users.js file only when the button is clicked using the import() function.
+
+This function takes the file name as paramenter and returns a priomise which is completed when the file is completly loaded.
+Then we can poppulate the data in our users array.
+
+This is the difference between require, import and dynamic import.
+
+---
+## Refferences -
```
Comment (code-reaper08) [inline Web_Development/Javascript/Require vs Import/README.md:63]: Typo

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Hello @HarjasSodhi 👋 
I've left a review.
Other than that everything looks cool 🆒

File: Web_Development/Javascript/Require vs Import/README.md (hunk 1 above)
Comment (HarjasSodhi) [reply Web_Development/Javascript/Require vs Import/README.md]: @code-reaper08 Thanks for the review . Made the changes

**Review by code-reaper08 — APPROVED** [review]
LGTM ✨

**Pull request merged**